src/style.css

```diff
@@ -1 +1,70 @@
 /* Styles go here */
+
+.body {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgb(0 0 0 / 5%);
+}
+
+.logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.nav-list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav-list-item {
+  margin-left: 20px;
+}
+
+.nav-list-item:first-child {
+  margin-left: 0;
+}
+
+.nav-list-item > a {
+  text-decoration: none;
+  position: relative;
+  display: flex;
+  align-items: center;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  height: 60px;
```

Comment:
I would use vertical paddings for this purpose

Reply:
next time, use paddings – it's not a good practice to hardcode such values like `height`